Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ghcr.io/pyvista/pyvista:latest
+MAINTAINER "PyVista Developers"
+SHELL ["/bin/bash", "-c"]
+
+ARG NB_USER=jovyan
+ARG NB_UID=1000
+ENV USER ${NB_USER}
+ENV NB_UID ${NB_UID}
+ENV HOME /home/${NB_USER}
+
+WORKDIR $HOME
```

Comment:
I believe all of this should already be set in the base image `jupyter/base-notebook:python-3.9.7` and `ghcr.io/pyvista/pyvista:latest`:
https://github.com/pyvista/pyvista/blob/9957f163861a0040f94c853efd32d0c02fd413c5/docker/jupyter.Dockerfile#L26